frontends/web/src/components/amount/conversion-amount.tsx

```diff
@@ -70,12 +69,8 @@ export const ConversionAmount = ({
             unit={conversionUnit}
           />
         </>
-      ) : (
-        <div className={styles.skeletonContainer}>
-          <Skeleton />
-        </div>
-      )}
-      <span className={styles.txUnit}>{' '}{conversionUnit}</span>
+      ) : null }
+      { (conversion || sendToSelf) ? <span className={styles.txUnit}>{' '}{conversionUnit}</span> : null }
```

Comment:
nit: I know we don't have prettier (yet), but it would help me reading it a lot by separating jsx and js with some newlines, it also makes a nicer git history in the future 
```suggestion
      { (conversion || sendToSelf) ? (
        <span className={styles.txUnit}>
          {' '}
          {conversionUnit}
        </span>
      ) : null }
```